components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/util/ActionExecutorConfig.java

```diff
@@ -65,6 +65,8 @@ public boolean isExecutionForActionTypeEnabled(ActionType actionType) {
         switch (actionType) {
             case PRE_ISSUE_ACCESS_TOKEN:
                 return isActionTypeEnabled(ActionTypeConfig.PRE_ISSUE_ACCESS_TOKEN.getActionTypeEnableProperty());
+            case AUTHENTICATION:
+                return isActionTypeEnabled(ActionTypeConfig.AUTHENTICATION.getActionTypeEnableProperty());
```

Comment:
Let's add the config to the identity.xml.j2 and the default.json as well.

Reply:
Addressed with commit 2791587f98eb05e7560e1de7950fb41ca23d1218